composer.json

```diff
@@ -18,7 +18,8 @@
     "friendsofphp/php-cs-fixer": "^1.11",
     "phpunit/phpunit": "^4.8.36|^5.7",
     "sebastian/comparator": ">=1.2.3",
-    "phpseclib/phpseclib": "^2"
+    "phpseclib/phpseclib": "^2",
+    "kelvinmo/simplejwt": "^0.2.5"
```

Comment:
Does `firebase/php-jwt` not have the utilities needed to support the new verification? Perhaps we could think about moving everything over? The firebase library leaves much to be desired.

Reply:
The firebase library does not support ES256. I submitted a PR which adds support, but it's pretty limited compared to the support afforded by these other libraries (https://github.com/firebase/php-jwt/pull/256)

The problem with switching over entirely is, in the case of `kelvinmo/simplejwt`, the additional extenion requirement  of `opeenssl` and `hash`, and also that it comes in a pre-1.0 version.

Reply:
Looking into it, `hash` is compiled into PHP by default, so it may be possible to submit a PR to the maintainer to remove the `openssl` extension requirement